build/build.xml

```diff
@@ -79,7 +79,7 @@
 
 		<echo message="Compiling tests..." />
 		<javac srcdir="${test.src}" destdir="${test.build}" source="${src.version}" target="${src.version}" includeantruntime="false" debug="${debug}" encoding="UTF-8">
-			<compilerarg value="-Xlint:all"/>
+			<!--<compilerarg value="-Xlint:all"/>-->
```

Comment:
Is this still necessary?

I thought this had been put in place as part of our CI badge?